### PR TITLE
add test for exporter metrics

### PR DESCRIPTION
**Pull request by gabemontero**

### CODE REVIEW

File: operator/test/manifests/test/metrics/curl-metrics-service-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+              script: |
+                stats="$(curl http://pipeline-metrics-exporter-service.openshift-pipelines.svc.cluster.local:9117/metrics 2> /dev/null | grep pipelinerun_duration)"
+                if [ -z "$stats" ]; then
+                   echo "FAILED:  did not get pipelinerun_duration stats"
+                   echo $stats
+                   exit 1
+                fi
+                echo "SUCCESS: got pipelinerun_duration stats:"
+                echo $stats
+
```
Comment (ramessesii2) [inline operator/test/manifests/test/metrics/curl-metrics-service-pipeline.yaml:26]: Static-code-analysis is frowning at this extra line.

**Review by ramessesii2 — APPROVED** [review]
Locally tested. LGTM

File: operator/test/manifests/test/metrics/curl-metrics-service-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        taskSpec:
+          steps:
+            - image: registry.access.redhat.com/ubi8/ubi-minimal:8.7
+              name: metrics-curl
+              securityContext:
+                runAsNonRoot: true
+              script: |
+                stats="$(curl http://pipeline-metrics-exporter-service.openshift-pipelines.svc.cluster.local:9117/metrics 2> /dev/null | grep pipelinerun_duration)"
+                if [ -z "$stats" ]; then
+                   echo "FAILED:  did not get pipelinerun_duration stats"
```
Comment (enarha) [inline operator/test/manifests/test/metrics/curl-metrics-service-pipeline.yaml:20]: nit: while fixing the extra line, here is an extra space after :

**Comment by gabemontero** [comment]
ok static code passing @ramessesii2 @enarha and I believe extra space is accounted for @enarha

**Comment by gabemontero** [comment]
tests are green

given @ramessesii2 and @enarha prior lgtm's I'm merging

**Pull request merged**